src/frontend/apps/desk/src/api/fetchApi.ts

```diff
@@ -31,7 +31,12 @@ export const fetchAPI = async (input: string, init?: RequestInit) => {
 
   // todo - handle 401, redirect to login screen
   // todo - please have a look to this documentation page https://mozilla-django-oidc.readthedocs.io/en/stable/xhr.html
-  response.status === 401 && logout();
+  if (response.status === 401) {
+    logout();
+    window.location.replace(
+      new URL('authenticate/', process.env.NEXT_PUBLIC_API_URL).href,
+    );
```

Comment:
Could we keep the logic of the authentication in the auth feature ? The function `logout` should do the redirection so, let's keep it DRY moreover,  we have this part in 2 different places, not easy for a future refacto.

Reply:
As I mentioned, I will handle this refacto, moreover, this code doesn't log out the user, but re-login her. Can totally encapsulate the navigation in the Auth feature.